modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectDefinitionsRelationshipsMVCResourceCommand.java

```diff
@@ -75,6 +75,15 @@ protected void doServeResource(
 					_portal.getCompanyId(resourceRequest), true,
 					WorkflowConstants.STATUS_APPROVED);
 		}
+		else if (FeatureFlagManagerUtil.isEnabled("LPS-167253")) {
+			objectDefinitions =
+				ListUtil.filter(
+					_objectDefinitionLocalService.getObjectDefinitions(
```

Comment:
@seltong we must create a new method that considers the system and modifiable parameters. Or just the modifiable (if that's the case).

P.S. 1: @rebsilva just sent a commit removing this FF https://github.com/liferay-objects/liferay-portal/pull/1945/commits/b713bdc544d89997fcebf57b5dcf4364231b18ed

P.S. 2: For this case, I believe we don't need to treat this, since now we can list all the objects.

Reach me on slack, please.